### PR TITLE
Implement DragonBallFaker

### PR DESCRIPTION
**Pull request by Rahtoken**
I have implemented the DragonBallFaker.
Please have a look.

### CODE REVIEW

File: tests/FakerDotNet.Tests/FakerTests.cs [hunk 1]
```diff
@@ -91,6 +91,11 @@ public void Date_returns_IDateFaker()
             Assert.IsInstanceOf<IDateFaker>(Faker.Date);
         }
 
+        [Test]
+        public void DragonBall_returns_IDragonBallFaker()
+        {
+            Assert.IsInstanceOf<IDragonBallFaker>(Faker.DragonBall);
+        }
```
Comment (mrstebo) [inline tests/FakerDotNet.Tests/FakerTests.cs]:
```suggestion
        }

```

Comment (mrstebo) [reply tests/FakerDotNet.Tests/FakerTests.cs]: Just an extra blank line here

File: src/FakerDotNet/Data/DragonBallData.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+
+namespace FakerDotNet.Data
+{
+    internal static class DragonBallData
+    {
+        public static readonly IEnumerable<string> Characters = new []
```
Comment (mrstebo) [inline src/FakerDotNet/Data/DragonBallData.cs]:
```suggestion
        public static readonly IEnumerable<string> Characters = new[] 
```

Comment (mrstebo) [reply src/FakerDotNet/Data/DragonBallData.cs]: To keep in line with the way the others define their arrays :wink:

**Comment by Rahtoken** [comment]
Thank you for the response. 
Could you please have another look?

**Comment by mrstebo** [comment]
Now it is just missing the documentation.

You need to create *docs/dragon_ball.md* and add a link to it in the readme.

After that I will merge it in :+1:

**Comment by Rahtoken** [comment]
Thank you so much for your patience!
I hope I have made all the necessary additions.

**Comment by mrstebo** [comment]
Nice job! Thanks for the contribution :tada: :beers:

**Pull request merged**